src/rshim_usb.c

```diff
@@ -75,23 +77,34 @@ static void rshim_usb_delete(rshim_backend_t *bd)
   free(dev);
 }
 
+static int chan_to_wValue(int chan, uint16_t ver_id)
```

Comment:
It's ok to use lower case as function name to be consistent.

Reply:
done.